src/bootstrap/bootstrap.py

```diff
@@ -18,13 +19,26 @@
 
 def get(url, path, verbose=False):
     print("downloading " + url)
-    # see http://serverfault.com/questions/301128/how-to-download
-    if sys.platform == 'win32':
-        run(["PowerShell.exe", "/nologo", "-Command",
-             "(New-Object System.Net.WebClient).DownloadFile('" + url +
-                "', '" + path + "')"], verbose=verbose)
-    else:
-        run(["curl", "-o", path, url], verbose=verbose)
+    for url, path in ((url, path), (url + ".sha256", path + ".sha256")):
+        # see http://serverfault.com/questions/301128/how-to-download
+        if sys.platform == 'win32':
+            run(["PowerShell.exe", "/nologo", "-Command",
+                "(New-Object System.Net.WebClient).DownloadFile('" + url +
+                    "', '" + path + "')"], verbose=verbose)
+        else:
+            run(["curl", "-o", path, url], verbose=verbose)
+
+def verify(path, verbose=False):
+    if verbose:
+        print("verifying " + path)
+    with open(path + ".sha256", "r") as dist:
+        expected, _ = dist.readline().split()
+    with open(path, "rb") as f:
+        found = hashlib.sha256(f.read()).hexdigest()
+    if found != expected:
+        if not verbose:
```

Comment:
Good eye, thanks. I must've missed renaming this one.

I think it makes sense to put this `RuntimeError` and the one in `run()` behind the `verbose` flag.